Resources/public/js/countryCodeChange.js

```diff
@@ -26,7 +26,9 @@ module.exports = {
                             $address.replaceWith(
                                 $(html).find('#' + id)
                             );
-                            $form.find('#' + id + '_countryCode').change(onCountryCodeChange);
+                            var $countryCode = $form.find('#' + id + '_countryCode');
```

Comment:
Can't we put this part mostly in the initialization in the js from the bundle?

Reply:
I'm not sure what you mean?

Reply:
Nevermind, I wasn't fully awak I think :D